data/supertypes.yml

```diff
@@ -50,3 +50,136 @@ user_journey_document_supertype:
         - taxon
         - topic
         - topical_event
+
+email_document_supertype:
+  name: "Email document type"
+  description: "High level group for email subscriptions use to identify publications and announcement"
+  default: other
+  items:
+    - id: "publications"
+      document_types:
+        - closed_consultation
+        - consultation
+        - consultation_outcome
+        - corporate_report
+        - correspondence
+        - decision
+        - foi_release
+        - form
+        - guidance
+        - impact_assessment
+        - independent_report
+        - international_treaty
+        - map
+        - national_statistics
+        - notice
+        - official_statistics
+        - open_consultation
+        - policy_paper
+        - promotional
+        - regulation
+        - research
+        - statistical_data_set
+        - statutory_guidance
+        - transparency
+    - id: "announcements"
+      document_types:
+        - announcement
+        - fatality_notice
+        - government_response
+        - news_story
+        - oral_statement
+        - press_release
+        - speech
+        - written_statement
+
+government_document_supertype:
+  name: "Government document type"
+  description: "Grouping for email subscriptions."
+  # we expect that everything should be mapped to a valid supertype.
+  default: other
+  items:
+    - id: Consultations
+      document_types:
+        - consultation
+    - id: Open consultations
```

Comment:
We might need sluggified id's for this, I'm not sure how spaces/capitalisation will work in rummager/other URLs.